metricflow/cli/main.py

```diff
@@ -213,6 +218,13 @@ def tutorial(ctx: click.core.Context, cfg: CLIContext, msg: bool, skip_dw: bool,
     default=False,
     help="In the query output, show the query that was executed against the data warehouse",
 )
+@click.option(
+    "--display-plans",
+    is_flag=True,
+    required=False,
+    default=False,
```

Comment:
don't need the `default`, it defaults to False already via the `is_flag`

Reply:
Updated.